e2e-tests/README.md

```diff
@@ -0,0 +1,11 @@
+# End to End Tests
```

Comment:
This is the only new file, everything else has just been moved around on the file system.